jekyll/_docs/ios-code-signing.md

```diff
@@ -41,7 +41,7 @@ The debug symbols file will be used by your distribution service of choice
 to produce call-stacks helping you to debug any crashes in your app that
 testers encounter.
 
-### OS X circle Keychain
+### OS X CircleCI Keychain
```

Comment:
this should still be `circle` as it's the name of a generated `circle` keychain.